src/functions-and-arrays.js

```diff
@@ -1,73 +1,558 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(arg1, arg2) {
+    if (arg1 > arg2) {
+        return arg1;
+    } else if (arg1 < arg2) {
```

Comment:
you could have done `else{ return arg 2}`
and saved some code with the else if and the other return :)